pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "HLTRho"
+version = "0.0.1"
+authors = [
+  { name="Alessandro Lupo", email="alessandro.lupo@cpt.univ-mrs.fr" },
+  { name="Niccolo' Forzano", email="2227764@swansea.ac.uk" },
```

Comment:
I'd recommend keeping your name consistent—in papers you've been using "Niccolò", so I'd recommend doing the same here.

Reply:
Implemented.